sw/device/lib/dif/dif_plic.c

```diff
@@ -16,7 +16,7 @@
 // If either of these static assertions fail, then the assumptions in this DIF
 // implementation should be revisited. In particular, `kPlicTargets`
 // may need updating,
-_Static_assert(RV_PLIC_PARAM_NUMSRC == 84,
+_Static_assert(RV_PLIC_PARAM_NUMSRC == 87,
```

Comment:
@silvestrst @lenary I don't have enough knowledge of PLIC DIF. Is this change enough?

Reply:
@eunchan i think i remember needing to change something else, otherwise the unit test failed.
let me see if i can find it.

Reply:
see here.. https://github.com/lowRISC/opentitan/blob/master/sw/device/tests/dif/dif_plic_unittest.cc#L81
This looks different from when I had to change it..so maybe see if unittests fail first?

Reply:
Thanks! That fix passes plic unittest.

Reply:
Late to the party, but yeah, essentially there's a unittest and the sanity test, if both pass then it should be all good!